hack/DebugWindowsNode.ps1

```diff
@@ -0,0 +1,80 @@
+Write-Output "Checking for common problems with Windows Kubernetes nodes"
+Î
+$global:DockerVersion = $null
+
+$currentVersion = Get-Item 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion'
+$OSProductName = $currentVersion.GetValue('ProductName')
+$OSBuildLabel = $currentVersion.GetValue('BuildLabEx')
+Write-Output "Container Host OS Product Name: $OSProductName"
+Write-Output "Container Host OS Build Label: $OSBuildLabel"
+
+Describe "Windows Version and Prerequisites" {
+    $buildNumber = (Get-CimInstance -Namespace root\cimv2 Win32_OperatingSystem).BuildNumber
+    It "Is Windows Server 2019" {
+        $buildNumber -eq 17763 | Should Be $true
```

Comment:
Can you make this check -ge 17763?
We support k8s all Windows server versions including/after Windows Server 2019.